app/controllers/admin/fields_definitions_controller.rb

```diff
@@ -76,8 +75,14 @@ def sort
 
   private
 
+  DEFAULT_PARAMS = %i[target label required hidden read_only choices_for_views].freeze
+
   def field_definition_params
-    params[:fields_definition] || {}
+    params.require(:fields_definition).permit(DEFAULT_PARAMS | %i[name])
```

Comment:
`field_definition_params` is also used in `#new`. I guess that `:fields_definition` will not be provided in such call so the `require` call will fail. Have you checked that?

btw, will it be safe to remove `attr_protected :account_id, :tenant_id` from `app/models/fields_definition.rb`?